js/src/Figure.ts

```diff
@@ -975,6 +980,7 @@ class Figure extends widgets.DOMWidgetView {
     y_padding_arr: any;
 
     private _update_requested: boolean;
+    private _relayout_requested: boolean;
```

Comment:
Maybe you need a default value of `false` here

Reply:
And I guess we don't need any underscore prefix in TypeScript. It is already defined as private.

Reply:
I would also be in favor of using camelCase from now on.